package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "ha-map-card",
+  "version": "0.12.0",
```

Comment:
VERSION should come from the tag (or the tag from here)

Reply:
I don't believe its possible to have a package.json file without a version set 🤔 
I'm pretty rusty on the FE so not sure what the normal approach is - re: is the release triggered by the NPM version, or is there a workflow setting that on release?

e.g.
https://github.com/piitaya/lovelace-mushroom/blob/main/package.json
https://github.com/MindFreeze/ha-sankey-chart/tree/master

Unsure what option is best TBH

Reply:
I'll introduce [devenv.sh](https://devenv.sh/) with a proper release flow picking the version from this file.